### PR TITLE
[mpfr] Remove docs

### PR DESCRIPTION
**Pull request by Thomas1664**
**Describe the pull request**

- #### What does your PR fix?
  Merge license files and fix copyright expression
  Remove docs

### CODE REVIEW

**Review by FrankXie05 — DISMISSED** [review]

**Review by JavierMatosD — APPROVED** [review]

File: ports/mpfr/vcpkg.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "mpfr",
   "version": "4.1.0",
-  "port-version": 4,
+  "port-version": 5,
   "description": "The MPFR library is a C library for multiple-precision floating-point computations with correct rounding",
   "homepage": "https://www.mpfr.org",
-  "license": "LGPL-3.0-or-later",
+  "license": "GPL-3.0-only OR LGPL-3.0-or-later",
```
Comment (JavierMatosD) [inline ports/mpfr/vcpkg.json:7]:
```suggestion
  "license": "LGPL-3.0-or-later",
```
LGPL implies GPL

**Review by JavierMatosD — CHANGES_REQUESTED** [review]

File: ports/mpfr/vcpkg.json (hunk 1 above)
Comment (FrankXie05) [reply ports/mpfr/vcpkg.json]: `LGPL-3.0-or-late` contains `GPL-3.0-only` and also provides a supplementary description of `GPL-3.0-only`.
See https://spdx.org/licenses/LGPL-3.0-or-later.html

**Review by FrankXie05 — APPROVED** [review]

**Review by JavierMatosD — APPROVED** [review]
Thanks!

**Comment by JavierMatosD** [comment]
Thanks!

**Pull request merged**